plugins/woocommerce/src/Admin/API/OnboardingProfile.php

```diff
@@ -449,6 +449,13 @@ public static function get_profile_properties() {
 				'readonly'          => true,
 				'validate_callback' => 'rest_validate_request_arg',
 			),
+			'plugins_page_skipped' => array(
```

Comment:
```suggestion
			'is_plugins_page_skipped' => array(
```
Nit: the other boolean params have an `is_` prefix

Reply:
Good call! Renamed it in [2486c2b](https://github.com/woocommerce/woocommerce/pull/38764/commits/2486c2b9e6fb2bdf68fdc280d209b3e95979faec)